tests/mul_test.c

```diff
@@ -0,0 +1,9 @@
+#include <stdio.h>
```

Comment:
What is the difference between this file and driver.sh:L324-331? Can we merge this test suite into driver.sh? I suppose there is something duplicate.